chanzuckerberg/corpora-data-portal
# thuang-130-CZIF-policy-change-for-real

https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell/130

#### Reviewers
**Functional:** 
@brianraymor 

**Readability:** 
@Bento007 

---

## Changes
1. Update `/tos` and `/privacy` to use the new CZIF content
2. Remove "updates coming" for both cookie banner and footer links

## Definition of Done (from ticket)
- [ ] Remove “(updates coming)” next to Terms of Service and Privacy Policy links on DP cookie banner.
- [ ] Remove “(updates coming)” next to Terms of Service and Privacy Policy links at the bottom of the portal pages.
- [ ] Remove cellxgene.cziscience.com/previewpolicies.
- [ ] Replace the contents of https://cellxgene.cziscience.com/tos/ with the TOS section from [this document](https://docs.google.com/document/d/1ya_mFaYSOSaJdXHcXHYpQfwOEVokNQN6yH8zYUuohfs/edit#). 
- [ ] Replace the contents of https://cellxgene.cziscience.com/privacy/ with the Privacy section from [this document](https://docs.google.com/document/d/1ya_mFaYSOSaJdXHcXHYpQfwOEVokNQN6yH8zYUuohfs/edit#). 

## QA steps (optional)
1. Visit the following links to make sure they meet the specs 🙏:
    1. https://thuang-czif-frontend.rdev.single-cell.czi.technology/
    2. https://thuang-czif-frontend.rdev.single-cell.czi.technology/tos/
    2. https://thuang-czif-frontend.rdev.single-cell.czi.technology/privacy/

Privacy:
<img width="1239" alt="Screen Shot 2021-03-22 at 3 41 28 PM" src="https://user-images.githubusercontent.com/6309723/112068502-47d41980-8b27-11eb-8f51-4785aba68241.png">

ToS:
<img width="1224" alt="Screen Shot 2021-03-22 at 3 41 41 PM" src="https://user-images.githubusercontent.com/6309723/112068506-486cb000-8b27-11eb-89a1-ceea77aa480d.png">

Cookie banner:
<img width="602" alt="Screen Shot 2021-03-22 at 3 43 21 PM" src="https://user-images.githubusercontent.com/6309723/112068508-49054680-8b27-11eb-8da5-8a340aa53652.png">

Footer:
<img width="505" alt="Screen Shot 2021-03-22 at 3 44 25 PM" src="https://user-images.githubusercontent.com/6309723/112068509-499ddd00-8b27-11eb-9855-45a9d2b0edae.png">
